libraries/ESP_I2S/src/ESP_I2S.h

```diff
@@ -17,10 +27,16 @@ class I2SClass: public Stream {
     I2SClass();
     ~I2SClass();
 
+    //STD + TDM mode
     void setPins(int8_t bclk, int8_t ws, int8_t dout, int8_t din=-1, int8_t mclk=-1);
     void setInverted(bool bclk, bool ws, bool mclk=false);
 
-    bool begin(uint32_t rate, i2s_data_bit_width_t bits_cfg, i2s_slot_mode_t ch);
+    //PDM TX + PDM RX mode
+    void setPinsPdmTx(int8_t clk, int8_t dout0, int8_t dout1);
+    void setPinsPdmRx(int8_t clk, int8_t din0, int8_t din1, int8_t din2, int8_t din3);
```

Comment:
Have `dout1` and `din1+` default to `-1`. They are not always required. Or provide different signatures, depending on the number of channels.

Reply:
done